config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+  end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :posts, only: [:index, :show] do
+  end
+
+
+  root 'users#index'
```

Comment:
- [ ] Kinldy as the previous reviewer show you, nest your `post` routes inside your `user` page. You can follow the example provided by him.